common_app/macros.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+/**
+ * Macro for the size of a specific structure field.
+ */
+#define MEMBER_SIZE(type, member) (sizeof(((type *) 0)->member))
```

Comment:
Maybe we can add:
`#define ARRAY_LENGTH(array) (sizeof((array))/sizeof((array)[0]))`